rat/grammar.ebnf

```diff
@@ -5,7 +5,14 @@
 
 start::Program = { statements+:statement }+ $;
 
-statement::Statement = left:factor ~ op:('~' | '=') ~ right:expression ';';
+statement =
+    | sample_statement
+    | assignment_statement
+;
+
+sample_statement::Statement = left:factor op:("~") ~ right:expression ';';
+
+assignment_statement::Statement = left:variable op:('=') ~ right:expression ';';
```

Comment:
The problem I see is using `variable` on LHS of statement may mean constraints are also legal? How would constraints on assigned parameters work? IIRC stan doesn't do any transformations, but just as a sanity check.